WalletWasabi.Fluent/Styles/Styles.axaml

```diff
@@ -5,7 +5,7 @@
     <sys:Double x:Key="IconElementThemeHeight">16</sys:Double>
     <Easing x:Key="FluentEasing">0.4,0,0.6,1</Easing>
     <KeySpline x:Key="FluentKeySpline">0.4,0,0.6,1</KeySpline>
-    <FontFamily x:Key="MonospacedFont">avares://WalletWasabi.Fluent/Assets/Fonts/CascadiaCode.ttf</FontFamily>
+    <FontFamily x:Key="MonospacedFont">avares://WalletWasabi.Fluent/Assets/Fonts#Cascadia Code</FontFamily>
```

Comment:
Just as a reminder... It was not worked because of my fault, we must find a new good looking monospaced font that matches the new modern UI.